cli/lsp/testing/definitions.rs

```diff
@@ -137,27 +138,21 @@ impl TestDefinitions {
     })
   }
 
+  pub fn inject(&mut self, data: lsp_custom::TestData) -> bool {
```

Comment:
some doc as to what this is any why

Reply:
Sure, though it's already clearer than before... we had `TestDefinitions::injected` already, I extracted a method to populate it. I suppose the return-if-insertion-succeeded thing could be doc'd